pkg/morph/client/util.go

```diff
@@ -157,6 +158,21 @@ func IntFromStackItem(param stackitem.Item) (int64, error) {
 	}
 }
 
+// BigIntFromStackItem receives numerical value from the value of a smart contract parameter.
+func BigIntFromStackItem(param stackitem.Item) (*big.Int, error) {
+	switch param.Type() {
+	case stackitem.IntegerT, stackitem.ByteArrayT:
+		i, err := param.TryInteger()
```

Comment:
Seems like we can do just
```
func BigIntFromStackItem(param stackitem.Item) (*big.Int, error) {
  return param.TryInteger()
}
```
This is at odds with the approach used in other functions, so it can be done separately in a complex.

Reply:
Seems like a good option! Thank you, fixed.